www/js/gameController.js

```diff
@@ -402,6 +403,21 @@ var gameController = {
 
     removeBasket: function () {
         this.player.destroy();
+    },
+
+    displayButtonToExitToMainMenu: function() {
+        if (game.paused){
```

Comment:
This shouldn't be necessary, right? We should already know the game is paused when this method is called.